src/backend/opengl/opengl_display.cpp

```diff
@@ -188,10 +187,14 @@ void c_OpenGLDisplay::OnPaint(wxPaintEvent&)
 {
     wxPaintDC dc(m_GLCanvas.get());
 
-    glBindFramebuffer(GL_DRAW_FRAMEBUFFER, 0);
-    const auto s = m_GLCanvas->GetSize();
-    glViewport(0, 0, s.GetWidth(), s.GetHeight());
+    SetGLContextOnGLCanvas();
```

Comment:
I'd rather not do it on each `OnPaint`, depending on driver+platform there could be an overhead. Is it perhaps sufficient on macOS to do it only on the first `OnPaint`?

Reply:
It looks like this has to be done on each OnPaint on macOS. I tried to wrap it within dispatch_once() and no luck. 

OnPaint event is the only event that I saw (so far) which has AppKit and QuartzCore symbols in the backtrace...